### ISSUE TITLE
[Bug][ally]:  Banner add border in WHCM/forced-colors: active

### ISSUE THREAD

**Issue by majornista** [adobe/spectrum-web-components#3347]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

Banner

### Expected behavior

Banner should have a border when its background-color is replaced with canvas color in Windows High Contrast Mode/forced-colors: active, so that it can be distinguished from other content.

### Actual behavior

Banner lacks a border when its background-color is replaced with canvas color in Windows High Contrast Mode/forced-colors: active.

### How can we reproduce this issue?

3. Scroll to '....'
4. Check console
5. See error

**Comment by Westbrook** [comment]
This pattern is no longer supported by Spectrum, we should likely deprecate/remove it from the project.

**Comment by najikahalsema** [comment]
We should come up with a process for deprecation. How do we want to let users know when something is being deprecated, and when do we want to remove the code marked for deprecation? @Rajdeepc @jnjosh

**Comment by Rajdeepc** [comment]
Banner is in process of deprecated as in #4176 . We will close this bug once the PR is merged.

**Comment by Rajdeepc** [comment]
Closing this as part of #4176
